data/data/ibmcloud/network/main.tf

```diff
@@ -54,11 +54,29 @@ module "cis" {
   cis_id         = var.ibmcloud_cis_crn
   base_domain    = var.base_domain
   cluster_domain = var.cluster_domain
+  is_external    = local.public_endpoints
```

Comment:
consider passing this in as a boolean to the ibm-variables where the logic is moved to tfvars in the installer.

Reply:
The boolean is based on PublishStrategy currently, and just passed around after that.
If it makes sense to add boolean version of the PublishStrategy (basically) to tfvars that gets used in place of that `Internel`/`External` string, I can do that in a follow up PR.